python/GafferImageUI/CatalogueUI.py

```diff
@@ -150,6 +150,7 @@ def __statusIconColumnValueProvider( image, catalogue ) :
 registerColumn( "status", "", __statusIconColumnValueProvider, ColumnType.Icon )
 registerColumn( "name", "Name", lambda image, _ : image.getName() )
 registerColumn( "frame", "Frame", contextValueProvider( "frame" ) )
+registerColumn( "description", "Description", lambda image, _ : image["description"].getValue() )
```

Comment:
`image["description"]` feeds into `catalogue["out"].metadata()["ImageDescription"]`. We would be better off reading from the latter, because that works for images that are loaded from disk too.

Reply:
Updated in https://github.com/GafferHQ/gaffer/pull/3688/commits/23cef312a7dba8351bf00bbdf68450d3e2ad12d6